cli/src/pcluster/aws/ec2.py

```diff
@@ -67,7 +71,20 @@ def get_default_instance_type(self):
     @AWSExceptionHandler.handle_client_exception
     def describe_subnets(self, subnet_ids):
         """Return a list of subnets."""
-        return list(self._paginate_results(self._client.describe_subnets, SubnetIds=subnet_ids))
+        result = []
```

Comment:
If we're going to add this extra behaviour we need to write uni test for this method.

Reply:
Added